### DISCUSSION TITLE
Information about RODA

### DISCUSSION THREAD

**Discussion by NikoBele1** [keeps/roda#2850] [Q&A] [answered]
Hi,

while looking for tools to create a large movie archive (longterm preservation) i found roda. I am looking for information about storage, but i can't find anything in the documentation. Where does the ingested data go? What storage options do i have? Can someone help me out and give some insight how this works with roda?

Greetings!

**Comment by luis100** [comment, marked as answer]
RODA uses the file system to store data, the information is under /roda/data/storage and is laid out in the E-ARK AIP standard specification.

![image](https://user-images.githubusercontent.com/631728/236518307-157f77d9-77dc-46ee-95d5-c8caca1fd438.png)

You can use any storage option that would allow you to mount as a file system, we normally use directly connected SAN storage, or network storage (like NFSv4). Object stores (like Amazon S3) are not currently directly supported, unless you install an additional components that would allow to mount them as file system (like Amazon's S3 File Gateway or [other options](https://bluexp.netapp.com/blog/amazon-s3-as-a-file-system)) but we have never tried them in production.

**Reply by NikoBele1** [reply]
Hey @luis100, thank you for the response. Where does /roda/data/storage reference to? I cant find that.

**Reply by luis100** [reply]
`/roda/data` is a path within the RODA container. It should be mapped it a persistent volume (local or remote).
If you are not using Docker to deploy RODA, then data is kept under RODA home folder, generally defined with RODA_HOME environment variable, by default it goes under the home of the user running RODA, under a `.roda` folder.

**Reply by NikoBele1** [reply]
oh my bad, i thought /roda/data/storage was a relative path to the documentation about storage. Thanks for the help!
